cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityUserAndRoleManagementTest.scala

```diff
@@ -1,486 +0,0 @@
-            | * For `SET PASSWORD`:
-            | ** The `password` can either be a string value or a string parameter.
-            | ** The optional `PLAINTEXT` in `SET PLAINTEXT PASSWORD` has the same behaviour as `SET PASSWORD`.
-            | ** The optional `ENCRYPTED` can be used to create a user when the plaintext password is unknown but the encrypted password is available
-            |    (e.g. from a database backup). In this case the password string is expected to be in the format of `<encryption-version>,<hash>,<salt>`.
-            | * If the optional `SET PASSWORD CHANGE [NOT] REQUIRED` is omitted then the default is `CHANGE REQUIRED`.
-            |   The `SET PASSWORD` part is only optional if it directly follows the `SET PASSWORD` clause.
-            | * The default for `SET STATUS` is `ACTIVE`.
-          """.stripMargin
-        )
-        p("For example, we can create the user `jake` in a suspended state, with the home database `anotherDb` and the requirement to change his password.".stripMargin)
-        query("CREATE USER jake SET PASSWORD 'abc' CHANGE REQUIRED SET STATUS SUSPENDED SET HOME DATABASE anotherDb", ResultAssertions(r => {
-          assertStats(r, systemUpdates = 1)
-        })) {
-          statsOnlyResultTable()
-          note {
-            p("[enterprise-edition]#The `SET STATUS {ACTIVE | SUSPENDED}` and `SET HOME DATABASE` parts of the command are only available in Enterprise Edition.#")
-          }
-        }
-        p("The created user will appear on the list provided by `SHOW USERS`.")
-        query("SHOW USERS YIELD user, suspended, passwordChangeRequired, roles, home WHERE user = 'jake'", assertUsersShown(Seq("jake"))) {
-          p(
-            """In this example we also:
-              |
-              |* Reorder the columns using a `YIELD` clause
-              |* Filter the results using a `WHERE` clause to show only the new user
-              |
-              |""".stripMargin
-          )
-          resultTable()
-        }
-        query("SHOW USERS YIELD roles, user WHERE \"PUBLIC\" IN roles RETURN user as publicUsers", ResultAssertions({ r: DocsExecutionResult =>
-          r.columnAs[String]("publicUsers").toSet should be(Set("neo4j", "jake"))
-          })) {
-          p(
-            """It is also possible to add a `RETURN` clause to further manipulate the results after filtering. In this
-              |case it is used to filter out the roles column and rename the users column to `publicUsers`.""".stripMargin)
-          resultTable()
-        }
-        note {
-          p(
-            """In Neo4j Community Edition there are no roles, but all users have implied administrator privileges.
-              |In Neo4j Enterprise Edition all users are automatically assigned the <<administration-security-roles-public, `PUBLIC`>> role, giving them a base set of privileges.""".stripMargin)
-        }
```

Comment:
This information is not explicit in the pages:

- https://neo4j.com/docs/cypher-manual/4.3/access-control/manage-users/
- https://neo4j.com/docs/cypher-manual/4.3/access-control/manage-roles/